mods/ra/maps/soviet-06b/soviet06b-AI.lua

```diff
@@ -0,0 +1,112 @@
+IdleHunt = function(unit) if not unit.IsDead then Trigger.OnIdle(unit, unit.Hunt) end end
+
+IdlingUnits = function()
+	local lazyUnits = enemy.GetGroundAttackers()
+
+	Utils.Do(lazyUnits, function(unit)
+		Trigger.OnDamaged(unit, function()
+			Trigger.ClearAll(unit)
+			Trigger.AfterDelay(0, function() IdleHunt(unit) end)
+		end)
+	end)
+end
+
+BaseBuildings =
+{
+	{ "apwr", CVec.New(5, -9), 500, true },
+	{ "tent", CVec.New(-4, -4), 400, true },
+	{ "proc", CVec.New(0, -8), 1400, true },
+	{ "weap", CVec.New(-4, -8), 2000, true },
+	{ "apwr", CVec.New(6, -5), 500, true }
+}
```

Comment:
Lua has "tables"; something like this would be more readable (removing obscure numbers):
```Lua
BaseBuildings =
{
	Apwr = { type = "apwr", pos = CVec.New(5, -9), cost = 500, exists = true },
	Barr = { type = "tent", pos = CVec.New(-4, -4), cost = 400, exists = true },
	Proc = { type = "proc", pos = CVec.New(0, -8), cost = 1400, exists = true },
	Weap = { type = "weap", pos = CVec.New(-4, -8), cost = 2000, exists = true },
	Apwr2 = { type = "apwr", pos = CVec.New(6, -5), cost = 500, exists = true }
}
```